src/main/java/tds/exam/results/mappers/TestMapper.java

```diff
@@ -38,4 +51,53 @@ private static String parseBankKeyFromId(final String itemId) {
 
         return itemId.split("-")[0];
     }
+
+    private static long parseLatestAcademicYear(final String academicYearString) {
+        // If this is a range date, pick the latest year
+        if (academicYearString.contains("-")) {
+            return Long.parseLong(academicYearString.split("-")[1]);
```

Comment:
The rule is defined on page 4 of the TRT Data Model document: http://www.smarterapp.org/documents/TestResults-DataModel.pdf

"The school year in which the test is
administered. For academic years that
span a calendar year this is the four digit
year-end. E.g. 2013 for 2012-2013"